examples/nrf_802154_delayed_echo_client/src/nrf_802154_delayed_echo_client.c

```diff
@@ -0,0 +1,235 @@
+/*
```

Comment:
It would be good to run uncrustify with driver's `.uncrustify.cfg` on `nrf_802154_delayed_echo_client.c`, `nrf_802154_delayed_echo_server.c`, `nrf_802154_echo_client.c`, `nrf_802154_echo_server.c` and `mpsl_utils.c` files.

Reply:
Done.